journey-builder-triggers/template.ts

```diff
@@ -0,0 +1,161 @@
+import { RecommendationsConfig, RecipeReferenceLookup, RecipeReference, recommend } from "recs";
+import { UserAttributeLookup, UserAttributeReference, UserSegmentLookup, UserSegmentReference } from "common";
+
+export class JourneyBuilderTriggersUtils {
+    static defaultCatalogObject = "Product";
+    static defaultCatalogObjectAttributes = ["id", "name", "imageUrl", "url", "price"];
+    static maxRecommendations = 8;
```

Comment:
minor: maybe change the format to follow the convention for constants?

```suggestion
    static DEFAULT_CATALOG_OBJECT = "Product";
    static DEFAULT_CATALOG_OBJECT_ATTRIBUTES = ["id", "name", "imageUrl", "url", "price"];
    static MAX_RECOMMENDATIONS = 8;
```